Documentation/Kubernetes.md

```diff
@@ -0,0 +1,36 @@
+# kubeadm
+
+For information on deploying flannel manually, using the Kubernetes installer toolkit kubeadm, see [Installing Kubernetes on Linux with kubeadm][kubeadm].
+
+NOTE: if `kubeadm` is being used then pass `--pod-network-cidr=10.244.0.0/16` to `kubeadm init` to ensure that the `podCIDR` is set.
+
+kubeadm has RBAC enabled by default so you must apply the `kube-flannel-rbac.yml` manifest as well as the `kube-flannel.yml` manifest.
+
+* kubectl apply -f kube-flannel-rbac.yml -f kube-flannel.yml
+
+If you didn't apply the `kube-flannel-rbac.yml` manifest, you'll see errors in your flanneld logs about failing to connect. 
+* `Failed to create SubnetManager: error retrieving pod spec...`
+
+It is safe to just apply the `kube-flannel-rbac.yml` manifest and flannel should start.
+* kubectl apply -f `https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel-rbac.yml`
```

Comment:
besides what zbwright said, this whole command should be in `code` format, not just the URL